modules/ROOT/partials/glossary.adoc

```diff
@@ -0,0 +1,116 @@
+[glossary]
+
+[[action]]$action:: Freeform description of an action.
+[[alias]]$alias:: An alias name.
+[[alloc]]$alloc:: Allocator name.
+[[allocType]]$allocType:: Allocation type.
+[[auth]]$auth:: Auth provider name.
+[[boltMsgLenLimit]]$boltMsgLenLimit:: Bolt message length limit.
+[[boltServerState]]$boltServerState:: Bolt server state.
+[[cause]]$cause:: Freeform description of what caused the error.
+[[cfgSetting]]$cfgSetting:: Configuration setting key, for example, `server.https.enabled`.
+[[clause]]$clause:: A clause, for example, `USE`.
```

Comment:
Up for discussion: currently we show all examples e.g. the USE here with the same formatting. As an option, we could do it with the formatting the user will actually get back, which is almost always the same for the same parameter. Some examples how this would look:

[[clause]]$clause:: A clause, for example, USE.

[[cmd]]$cmd:: Cypher administrative command, for example, 'DROP DATABASE'.

[[db]]$db:: Database name, for example, \`myDb\`.

Reply:
At the moment, this is not consistent in the code. To be honest, I think we should leave the formatting to the docs and remove all unnecessary quotes, backticks, etc., from the code.